pkg/trace/pb/stats.proto

```diff
@@ -1,16 +1,15 @@
 syntax = "proto3";
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
-
 package pb;
+option go_package="github.com/DataDog/datadog-agent/pkg/trace/pb";
```

Comment:
The `.pb.go` file associated with this `.proto` file has not been regenerated as part of this PR. I am going to move this `.proto` file change into another PR as it requires extensive changes to helper functions, tests, etc.  Ignore this file for now.